generators/app/tasks/appSetup/loginFeatureFiles.js

```diff
@@ -1,33 +1,38 @@
 module.exports = function loginFeatureFiles() {
-  // src/redux/authHandlers.js
+  // src/redux/auth/reducer.js
   this.fs.copy(
-    this.templatePath('src', 'redux', 'authHandlers.js'),
-    this.destinationPath(this.projectName, 'src', 'redux', 'authHandlers.js')
+    this.templatePath('src', 'redux', 'auth', 'reducer.js'),
+    this.destinationPath(this.projectName, 'src', 'redux', 'auth', 'reducer.js')
+  );
+  // src/redux/auth/actions.js
+  this.fs.copy(
+    this.templatePath('src', 'redux', 'auth', 'actions.js'),
+    this.destinationPath(this.projectName, 'src', 'redux', 'auth', 'actions.js')
   );
   // src/services/AuthService.js
   this.fs.copy(
     this.templatePath('src', 'services', 'AuthService.js'),
     this.destinationPath(this.projectName, 'src', 'services', 'AuthService.js')
   );
-  // src/screens/home/HomeContainer.js
+  // src/screens/home/layout.js
   this.fs.copyTpl(
-    this.templatePath('src', 'screens', 'home', 'HomeContainer.js'),
-    this.destinationPath(this.projectName, 'src', 'screens', 'home', 'HomeContainer.js'),
+    this.templatePath('src', 'app', 'screens', 'home', 'layout.ejs'),
+    this.destinationPath(this.projectName, 'src', 'app', 'screens', 'home', 'layout.js'),
```

Comment:
The index file is already created by the task baseFiles. Check https://github.com/Wolox/wolmo-bootstrap-react-native/pull/22/files#diff-ac4bd3b001c6b6a36df8643f2a6b7ec3R69